etc/kayobe/dnf.yml

```diff
@@ -200,7 +200,7 @@ rocky_9_gpg_key: "https://dl.rockylinux.org/pub/rocky/RPM-GPG-KEY-Rocky-9"
 
 # Whether to create a repo file for ELRepo. This affects RedHat-based
 # systems only.
-dnf_install_elrepo_9: false
+dnf_install_elrepo_9: true
```

Comment:
Are you sure you want to change the default here?

Reply:
Agreed, elrepo is known as a bit of a wild west of packages. Were you installing something specific from there?

Reply:
This change is not related to 9.3 upgrade, sorry. Please see my comment below.